src/main/scala/fi/oph/koski/raportit/PaallekkaisetOpiskeluoikeudet.scala

```diff
@@ -305,14 +305,14 @@ object PaallekkaisetOpiskeluoikeudet extends Logging {
       case (_, ("preiboppimaara", _)) => t.get("raportti-excel-default-value-ibsuoritus")
       case (_, (suorituksenTyyppi, koodiarvo))
         if InternationalSchoolOpiskeluoikeus.onLukiotaVastaavaInternationalSchoolinSuoritus(suorituksenTyyppi, koodiarvo) => t.get("raportti-excel-default-value-intschoollukio")
-      case (acc, (_, _)) if acc == t.get("raportti-excel-default-value-intschoollukio") => acc
-      case (_, (suorituksenTyyppi, koodiarvo))
-        if InternationalSchoolOpiskeluoikeus.onPeruskouluaVastaavaInternationalSchoolinSuoritus(suorituksenTyyppi, koodiarvo) => t.get("raportti-excel-default-value-intschoolperusopetus")
       // TODO: TOR-1685: Varmista, että nämä ovat oikein ESH:ssa, kun tietomalli on valmis:
       case (_, ("europeanschoolofhelsinkivuosiluokkanursery", _)) => t.get("raportti-excel-default-value-eshesiopetus")
       case (_, ("europeanschoolofhelsinkivuosiluokkaprimary", _)) => t.get("raportti-excel-default-value-eshperusopetus")
       case (_, ("europeanschoolofhelsinkivuosiluokkasecondarylower", _)) => t.get("raportti-excel-default-value-eshperusopetus")
       case (_, ("europeanschoolofhelsinkivuosiluokkasecondaryupper", _)) => t.get("raportti-excel-default-value-eshlukio")
+      case (acc, (_, _)) if acc == t.get("raportti-excel-default-value-intschoollukio") => acc
```

Comment:
tässä on joku rebase tms. varmaan järjestänyt koodin erikoisesti: parempi pitää international schoolit keskenään vierekkäin